File: athrill-account-client/projects/at-models/src/lib/users/user-role-link.model.ts
import { UserRoleModel } from './user-role.model';
import { UserModel } from './user.model';

export class UserRoleLinkModel{
    
    public UserRoleLinkId : number;
    public UserId : number;
    public UserRoleId: number; 

    public User: UserModel; 
    public UserRole: UserRoleModel; 
}

File: athrill-account-server/Libraries/Model/Users/UserRoleLinkModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.Users
{
    public class UserRoleLinkModel : BaseModel
    {
        public int UserRoleLinkId { get; set; }
        public int UserId { get; set; }
        public int UserRoleId { get; set; }

        #region links
        public  UserModel User { get; set; }
        public  UserRoleModel UserRole { get; set; }
        #endregion

        #region override
        public override int Id => UserRoleLinkId;
        #endregion
    }
}
